config/assets.go

```diff
@@ -84,11 +86,16 @@ func (a Assets) GetOppositeAsset(sourceChainId uint64, targetChainId uint64, ass
 
 }
 
+func (a Assets) GetWrappedToNativeNetwork(wrappedAsset string) int64 {
+	return a.wrappedToNativeNetwork[wrappedAsset]
+}
+
 func LoadAssets(networks map[int64]*parser.Network) Assets {
 	nativeToWrapped := make(map[int64]map[string]map[int64]string)
 	wrappedToNative := make(map[int64]map[string]*NativeAsset)
 	fungibleNetworkAssets := make(map[int64][]string)
 	fungibleNativeAssets := make(map[int64]map[string]*NativeAsset)
+	wrappedToNativeNetwork := make(map[string]int64)
```

Comment:
nit: if by any chance you've got wrapped assets on different networks, but with the same addresses, the last will override the previous.